### ISSUE TITLE
GitHub Enterprise Support Missing for Social Auth

### ISSUE THREAD

**Issue by mandeepbal** [getsentry/sentry#5758]
Hi, 
I don't see support for GitHub enterprise auth in on-premise Sentry. In previous versions of sentry I was able to use (https://github.com/getsentry/sentry-auth-github) but it has not been updated in over a year. I see a PR open since April for this support. However, it looks like the PR is stuck and nobody has mentioned how to get it merged (https://github.com/getsentry/sentry/pull/5068). Could someone please help review the PR?

**Comment by servomac** [comment]
Maybe [sentry-plugins](https://github.com/getsentry/sentry-plugins#github) could be useful.

**Comment by mandeepbal** [comment]
Thanks, but after reviewing it looks like that plugin doesn't support github enterprise either.

**Comment by dcramer** [comment]
We don't support GitHub enterprise at this time. At some point it will likely happen but it's not a priority for SSO (which is not the same thing) and outside of that were blocked on some internal refactor.
